app/controllers/concerns/generic_crud.rb

```diff
@@ -0,0 +1,61 @@
+module GenericCrud
+  extend ActiveSupport::Concern
+
+  def index
+    @entities = klass.includes(:answers, :comments).page(page).per(limit)
```

Comment:
need to fix this `includes`

Reply:
move this logic to the serializer and see what happen and how database queries are made 
read about( n + 1) bug issue in rails

Reply:
> how database queries are made read about( n + 1) bug issue in rails

I already faced it. It was generating a query for fetching each answer separately.